.github/workflows/e2e.yaml

```diff
@@ -16,10 +16,16 @@ jobs:
       matrix:
         globalnet: ['', '--globalnet']
         deploytool: ['operator', 'helm']
-        cable_driver: ['', 'wireguard']
+        cable_driver: ['', 'wireguard', 'libreswan']
         exclude:
+# Our Helm setup doesn’t know how to deploy other cable drivers
           - deploytool: 'helm'
             cable_driver: 'wireguard'
+          - deploytool: 'helm'
+            cable_driver: 'libreswan'
+# TODO: Remove exclusion when issue #552 is fixed
+          - globalnet: '--globalnet'
+            cable_driver: 'wireguard'
```

Comment:
```suggestion
```
The issue was fixed by:
    https://github.com/submariner-io/submariner/pull/562

Now we have an all-together different issue because the azure-ubuntu image has broken headers for wireguard... :/ https://github.com/submariner-io/submariner/issues/607

But I think it's unrelated to your patch, let's disable globally, we can ignore those runs for your patch.

Reply:
Done, I didn’t notice that that part had gone.